.travis.yml

```diff
@@ -11,14 +11,20 @@ php:
   - 7.3
   - 7.4snapshot
 
+env:
+  matrix:
+    - DRIVER="phpdbg"
```

Comment:
Can't use `pcov` yet as 

* we currently support PHP 7.1 and thus are limited to using `phpunit/phpunit:^7`, which depends on `phpunit/php-code-coverage:^6`
* `pcov` is supported via `phpunit/php-code-coverage:7.0.0` (https://github.com/sebastianbergmann/php-code-coverage/blob/master/ChangeLog.md#700---2019-02-01)